RealmCrustTests/ArrayAdaptorTests.swift

```diff
@@ -53,4 +52,5 @@ class ArrayAdaptorTests: RealmMappingTest {
         XCTAssertTrue(employeeStub.matches(object: obj[0]))
         XCTAssertTrue(employeeStub2.matches(object: obj[2]))
     }
+ */
```

Comment:
Oh ya, I think I should delete all this array adaptor/array mapping shit now that a `Mapping` can have a `SequenceKind` and all the mapping to an array falls through from that 🗑 Will do in the next pr to not overload this.